src/comparators.js

```diff
@@ -1,3 +1,5 @@
+import store from "@/store/store";
```

Comment:
Importing the store here doesn't feel very nice, this should be a utility file. What is wrong with always passing the albums as a param?

Reply:
This was mostly to avoid that we have to write `sort((t1, t2) -> compareTracks(t1, t2, rootState))` and to make the sorting function unaware of the structure of our store.
I've rewritten this so that we don't have to import the store here.